### PR TITLE
Added copy method for DBN

### PR DESCRIPTION
**Pull request by pdoongarwal**
Copy method for DynamicBayesianMethod is added, including its tests.

### CODE REVIEW

File: pgmpy/tests/test_models/test_DynamicBayesianNetwork.py [hunk 1]
```diff
@@ -136,6 +136,18 @@ def test_moralize(self):
...
+        self.network_copy = self.network.copy()
+        self.assertListEqual(self.network.nodes(), self.network_copy.nodes())
+        self.assertListEqual(self.network.edges(), self.network_copy.edges())
+        self.assertListEqual(self.network.get_cpds(), self.network_copy.get_cpds())
+        self.network_copy.add_node('Z') 
+        self.network.add_node('A')
+        self.network_copy.add_edge(('Z', 0), ('D', 0))
+        self.network.add_edge(('A', 0), ('D', 0))
+        self.assertNotEqual(self.network.nodes(), self.network_copy.nodes())
+        self.assertNotEqual(self.network.edges(), self.network_copy.edges())
```
Comment (ankurankan) [inline pgmpy/tests/test_models/test_DynamicBayesianNetwork.py:156]: @pdoongarwal Need more tests. Check other PRs for copy methods.

**Comment by pdoongarwal** [comment]
@ankurankan Will do ASAP!

**Comment by ankurankan** [comment]
@pdoongarwal Don't merge dev into your branch. Rather rebase your changes on the current dev.

**Comment by ankurankan** [comment]
@pdoongarwal What's the status of this ?

**Comment by pdoongarwal** [comment]
@ankurankan I have written the tests, just making sure all the tests are written.
Will send the final PR by tomorrow.

**Comment by khalibartan** [comment]
@ankurankan Failing continuous integration is not in regards with PR. Re-run the travis, tests will pass.

**Comment by yashu-seth** [comment]
@pdoongarwal What is the status of this?

**Comment by pdoongarwal** [comment]
I have added the copy method including tests.
I have also send the PR, you can merge it.
@yashu-seth Should I add anything else?

**Comment by ankurankan** [comment]
@pdoongarwal You need to add more tests. Check the tests of other `copy` methods.

**Comment by pdoongarwal** [comment]
@ankurankan Currently I am busy, please assign it to someone else.

**Pull request closed without merging**